locale/en-US/add-on.properties

```diff
@@ -28,13 +28,15 @@ sign_in_again_use_as_guest_button2=Use {{clientSuperShortname}} as a Guest
 panel_browse_with_friend_button=Browse this page with a friend
 panel_disconnect_button=Disconnect
 
-## LOCALIZATION_NOTE(first_time_experience_subheading2): Message inviting the
+## LOCALIZATION_NOTE(first_time_experience_subheading2, first_time_experience_subheading_button_above): Message inviting the
 ## user to create his or her first conversation.
 first_time_experience_subheading2=Click the Hello button to browse Web pages with a friend.
+first_time_experience_subheading_button_above=Click the button above to browse web pages with a friend.
```

Comment:
OK, re-reading the bug, it's possible that this needs to stay, though I don't think so.  So let's keep it to be safe.

However, s/Click the button/Click on the button/ and change web to Web as per the text in the bug.